### ISSUE TITLE
Pressing ESC to close interface does not get rid of overlay

### ISSUE THREAD

**Issue by Haftago** [fatfingers23/runelite-osrs-clanmate-export#7]
I and a few others use this a lot for 3rd party EHB/EHP tracking, current right now it is extremely annoying that pressing ESC to close interface does not get rid of the overlay, you HAVE to press the ❌ to get rid of overlay, or turn off plugin.

**Comment by fatfingers23** [comment]
Always happy to hear about it getting Used! Just to make sure are you talking about the instructions that show up? I've notice they've gotten a bit broken and do not always close. Would a config option to turn it off completely work too you think?

**Comment by Haftago** [comment]
Yeah only needed it for first time use honestly. Option to turn off would be amazing.

**Comment by fatfingers23** [comment]
I got a pr submitted for this! I will close this issue when it's merged. I added an option to stop showing the info blur "Click on members on the left." But mostly re-wrote it to use the in-game chat menu options to communicate with users. Thanks again for using the plugin and creating an issue!

<img width="802" alt="image" src="https://user-images.githubusercontent.com/20476157/170856540-9ed43c21-dc0e-423f-b018-b272252d5789.png">


https://github.com/runelite/plugin-hub/pull/2761
